README.Rmd

```diff
@@ -52,13 +52,13 @@ elmer supports a wide variety of model providers:
 
 ## Model choice
 
-If you're using elmer inside your organisation, you'll be limited to what your org allows, which is likely to be one provided by a big cloud provider, e.g. `chat_azure()`, `chat_bedrock()`, `chat_databricks()`, or `chat_snowflake()`. If you're using elmer for your own personal exploration, you have a lot more freedom so we have a few recommendations to help you get started:
+If you're using elmer inside your organisation, you'll be limited to what it allows, which is likely to be one provided by a big cloud provider, e.g. `chat_azure()`, `chat_bedrock()`, `chat_databricks()`, or `chat_snowflake()`. If you're using elmer for your own exploration, you'll have a lot more freedom, so we have a few recommendations to help you get started:
```

Comment:
I didn't use "it" here because I thought it might be unclear whether it referred to elmer or your org.

Reply:
I'm trying not to say organization more than once. Proximity and the "which" clause reinforces the idea that it's the organization but I guess it could be clearer. How about "what they allow"?

Reply:
What about "what your IT department allows"?